src/pyfmi/__init__.py

```diff
@@ -34,20 +34,6 @@
 int = np.int32
 np.int = np.int32
 
-def testattr(**kwargs):
```

Comment:
unused except for one case; one test marked for windows, but works for linux anyways.